src/main/java/org/jitsi/videobridge/ratecontrol/LastNBitrateController.java

```diff
@@ -573,7 +573,7 @@ private void add(long time, long rate)
         /**
          * Removes values added before <tt>time - period</tt>.
          */
-        private void clean(long time)
+        private synchronized void clean(long time)
```

Comment:
Thanks Lyubo! I don't consider the implementation of this class complete. I'll add your specific notes as comments in the code, to be addressed when we next work on it.